I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/CaveScene1.rpy", line 1264, in script
    menu:
  File "game/CaveScene1.rpy", line 1267, in <module>
    "Look at Vina Plant." if LookAtVine == Flase:
NameError: name 'Flase' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/CaveScene1.rpy", line 1264, in script
    menu:
  File "renpy/ast.py", line 1642, in execute
    choice = renpy.exports.menu(choices, self.set, args, kwargs, item_arguments)
  File "renpy/exports.py", line 1004, in menu
    condition = renpy.python.py_eval(condition)
  File "renpy/python.py", line 2249, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "renpy/python.py", line 2242, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/CaveScene1.rpy", line 1267, in <module>
    "Look at Vina Plant." if LookAtVine == Flase:
NameError: name 'Flase' is not defined

Windows-10-10.0.19041
Ren'Py 7.4.3.1414
 
Wed Apr  7 19:19:38 2021
